app/controllers/idv/cancellations_controller.rb

```diff
@@ -33,6 +35,31 @@ def destroy
 
     private
 
+    def barcode_step?
+      params[:step] == 'barcode'
+    end
+
+    def enrollment
+      InPersonEnrollment.where(user_id: current_user.id).first
```

Comment:
Can a user have multiple enrollments? Is this guaranteed to be the one created during the current in-person proofing session?

Reply:
LG-8423 says we're tracking activity when & after the cancel link is clicked on the barcode page. If the user's made it to the barcode page, their enrollment should be pending. So you should be able to get the correct enrollment like
```ruby
    def enrollment
      current_user.pending_in_person_enrollment
    end
```

Reply:
if we care about cancelations earlier in the process it could look something like
```ruby
    def enrollment
      current_user.establishing_in_person_enrollment || current_user.pending_in_person_enrollment
    end
```

Reply:
good point, i'll update to using the pending enrollment attached to the current user